setup.py

```diff
@@ -87,6 +87,17 @@ def package_tree(pkgroot):
     platforms='any',
     provides=['vispy'],
     install_requires=['numpy'],
+    extras_requires={
+        'ipython-static': ['ipython'],
+        'ipython-vnc': ['ipython>=2'],
+        'ipython-webgl': ['ipython>=2', 'tornado'],
+        'pyglet': ['pyglet>=1.2'],
+        'pyqt4': [],  # Why is this on PyPI, but without downloads?
```

Comment:
But it provides no benefit since `pip` fails to download anything. Unless it's some kind of indicator that you should install it by hand, though I don't see how that would be clear.